app/src/components/Dashboard/Navbar.tsx

```diff
@@ -147,6 +148,39 @@ export function Navbar({
           <Menu>
             <MenuButton as={IconButton} icon={<SettingsIcon />} size="md" />
             <MenuList>
+              {!isPremium && (
+                <MenuItem
+                  onClick={() => {
+                    onRefresh();
```

Comment:
This is wrong onClick I think. Do you want? Or open the stripe page?
```suggestion
      ipcRenderer.invoke('active-license');
```

Reply:
Oh oops good catch, this needs to open modal